packages/server/api.js

```diff
@@ -467,6 +520,8 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
         return
       }
 
+      await handleUnmentionsInMessage(oldMessage, text)
```

Comment:
Does this remove old mentions from the message's database document? (For example if I edit from `<@a> <@b>` to `<@a>`, does `b` get removed from the document?)

Reply:
As far as I know, yes.